ui/collapsible-panel.info/sample/main.reel/main.html

```diff
@@ -0,0 +1,285 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/html">
+<head>
+    <meta http-equiv="content-type" content="text/html; charset=utf-8">
+    <title>Collipsible Panel Example</title>
+    <link rel="stylesheet" type="text/css" href="main.css">
+    <script type="text/montage-serialization">
+        {
+            "owner": {
+                "properties": {
+                    "element": {"#": "main"}
+                }
+            },
+            "selectionController": {
+                "prototype": "montage/core/selection-controller",
+                "properties": {
+                    "isMultipleSelectionAllowed": false,
+                    "selection":[2]
+                }
+            },
+            "collapsiblePanel1": {
+                "prototype": "montage/ui/collapsible-panel.reel",
+                "properties": {
+                    "element": {"#": "collapsiblePanel-1"},
+                    "data": 1
+                },
+                "bindings": {
+                    "isExpanded": {"<->":"@selectionController.selection.has(@collapsiblePanel1.data)"}
+                }
+            },
+            "header1":{
+                "prototype": "montage/ui/text.reel",
+                "properties": {
+                    "element": {"#": "header-1"},
+                    "value":"Content1 Title"
+                }
+            },
+            "content1":{
```

Comment:
content1 -> body1 to match the parameter names, and the same for all the other body components in this HTML.